python/modules/IcePy/Proxy.cpp

```diff
@@ -2249,8 +2249,7 @@ IcePy::createProxy(const Ice::ObjectPrx& proxy, const Ice::CommunicatorPtr& comm
 {
     if (!type)
     {
-        PyTypeObject* proxyType = &ProxyType; // Necessary to prevent GCC's strict-alias warnings.
-        type = reinterpret_cast<PyObject*>(proxyType);
+        type = lookupType("Ice.ObjectPrx");
```

Comment:
Ice.ObjectPrx is the public base proxy type, it delegates to the parent class IcePy.ObjectPrx.